site/en/guides/border-router/tools.md

```diff
@@ -0,0 +1,244 @@
+# Tools and Scripts
+
+OpenThread Border Router (OTBR) includes a variety of tools and scripts that can
+be used for testing purposes.
+
+## PSKc generator
+
+`pskc` generates a Pre-Shared Key for the Commissioner (PSKc). The
+PSKc is used to authenticate an external Thread Commissioner to a Thread
+network. Build and install OTBR to use this tool.
+
+After building, `pskc` is located at
+[`/ot-br-posix/tools`](https://github.com/openthread/ot-br-posix/tree/master/tools/README.md).
+
+### Parameters
+
+The PSKc is generated from the following parameters:
+
+*   Commissioner Credential
+*   Thread Network Extended PAN ID
+*   Thread Network Name
+
+> Note: The Commissioner Credential is a user-defined string between 6 and 255 characters, UTF-8 encoded.
+  
+### Usage
+
+Syntax:
+
+```
+pskc {&lt;commissioner-credential&gt; &lt;extpanid&gt; &lt;network-name&gt;}
```

Comment:
Each parameter should be a replaceable variable

pskc {commissioner-credential} {extpanid} {network-name}

Reply:
Remove the &lt; and &gt; it should just be

pskc {commissioner-credential} {extpanid} {network-name}